src/modules/0.0.22/aws_ecs_simplified/sql/after_install.sql

```diff
@@ -75,20 +75,20 @@ BEGIN
   IF NEW.repository_uri IS NULL AND NEW.image_digest IS NULL THEN
     INSERT INTO repository (repository_name) VALUES (NEW.app_name || '-repository');
     -- fill in repository_name in container_definition
-    INSERT INTO container_definition ("name", essential, repository_name, task_definition_id, memory_reservation, host_port, container_port, protocol, log_group_name, env_variables)
+    INSERT INTO container_definition ("name", essential, repository_name, task_definition_id, memory_reservation, host_port, container_port, protocol, log_group_id, env_variables)
     VALUES (
       NEW.app_name || '-container', true,
       NEW.app_name || '-repository',
-      (SELECT id FROM task_definition WHERE family = NEW.app_name || '-td' AND status IS NULL LIMIT 1), get_mem_from_cpu_mem_enum(NEW.cpu_mem), NEW.app_port, NEW.app_port, 'tcp', NEW.app_name || '-log-group', NEW.env_variables
+      (SELECT id FROM task_definition WHERE family = NEW.app_name || '-td' AND status IS NULL LIMIT 1), get_mem_from_cpu_mem_enum(NEW.cpu_mem), NEW.app_port, NEW.app_port, 'tcp', (SELECT id from log_group WHERE log_group_name = NEW.app_name || '-log-group'), NEW.env_variables
```

Comment:
Same, in this file, all these queries should be by group name **and** region. I'd leave as a TODO for now